Sources/UIKit/Renderers/CollectionRender.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  PortalCollectionView.swift
+//  PortalView
+//
+//  Created by Argentino Ducret on 4/4/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import UIKit
+
+internal struct CollectionRenderer<MessageType>: UIKitRenderer {
+    
+    let properties: CollectionProperties<MessageType>
+    let style: StyleSheet<EmptyStyleSheet>
+    let layout: Layout
+    
+    func render(with layoutEngine: LayoutEngine, isDebugModeEnabled: Bool) -> Render<MessageType> {
+        let collectionView = PortalCollectionView(items: properties.items, layoutEngine: layoutEngine, layoutValues: properties.layoutValues)
```

Comment:
After making the change I've suggested regarding the layout properties. Here I would create a CollectionViewFlowLayout based on those properties pass it as a parameter to `PortalCollectionView`